frontend/styles.css

```diff
@@ -58,4 +58,24 @@
   #authpages #footerSignup {
     @apply hidden;
   }
+  /* Dark mode scrollbar */
+  .dark ::-webkit-scrollbar {
+    @apply bg-gray-600;
+  }
+  .dark ::-webkit-scrollbar-track {
+    @apply bg-gray-700;
+  }
+  .dark ::-webkit-scrollbar-thumb {
+    @apply bg-gray-600 w-5 h-5;
+  }
+  .dark ::-webkit-scrollbar-button:start:decrement {
+    @apply bg-gray-700;
+    background-image: url('data:image/svg+xml;charset=UTF-8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill="%23808080" stroke="%23808080" stroke-width="2" d="M6 15l6-6 6 6"/></svg>');
+    background-position: center;
```

Comment:
I'm fine with this but you might want to try `@apply object-center` and *see* if it works.  I'm not sure.  
https://tailwindcss.com/docs/object-position